packages/plugin-gate/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@datadog/datadog-ci-plugin-gate",
+  "version": "0.0.1",
```

Can you add this? (I'll update the How-to guide)

```suggestion
  "version": "0.0.1",
  "license": "Apache-2.0",
  "homepage": "https://github.com/DataDog/datadog-ci/tree/master/packages/plugin-gate",
  "repository": {
    "type": "git",
    "url": "https://github.com/DataDog/datadog-ci.git",
    "directory": "packages/plugin-gate"
  },
```